sechub/apply.go

```diff
@@ -188,14 +195,21 @@ func (sh *SecHub) Apply(ctx context.Context, cfg aws.Config, reason string) erro
 					if !ok {
 						return fmt.Errorf("not found: %s", fg.ControlID)
 					}
+
+					findingFilters := &types.AwsSecurityFindingFilters{
+						AwsAccountId: []types.StringFilter{types.StringFilter{Comparison: types.StringFilterComparisonEquals, Value: aws.String(a.AccountID)}},
+						ResourceId:   []types.StringFilter{types.StringFilter{Comparison: types.StringFilterComparisonEquals, Value: aws.String(r.Arn)}},
+						ProductName:  []types.StringFilter{types.StringFilter{Comparison: types.StringFilterComparisonEquals, Value: aws.String("Security Hub")}},
+						RecordState:  []types.StringFilter{types.StringFilter{Comparison: types.StringFilterComparisonEquals, Value: aws.String("ACTIVE")}},
+					}
+					if ctrlfg == "SECURITY_CONTROL" {
```

Comment:
How about using const as is without casting to string?

https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/securityhub@v1.31.0/types#ControlFindingGenerator

Also, how about using consts (ControlFindingGeneratorStandardControl, ControlFindingGeneratorSecurityControl) to do conditional branching with a switch statement, and then raise an error in the default: clause?

Reply:
Of course, after the merge, I can modify it.

Reply:
@k1LoW 
Thank you for the review. 
I have made the suggested changes.